mousestyles/GLRT_distribution.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import print_function, absolute_import, division
+
+import numpy as np
+from mousestyles import data
+
+
+def powerlaw_pdf(x, a):
+    """
+    The probability density function of truncated power law.
+
+    Parameters
+    ----------
+    x : int
+        x in formula p(x) = (alpha-1)*x^(-alpha).
+    a : int>1
+        alpha in formula p(x) = (alpha-1)*x^(-alpha).
+
+    Returns
+    -------
+    probability density : a float number
+        The probability density of power law at x.
+
+    Examples
+    --------
+    >>> powerlaw_pdf (2,2)
+    0.25
+    """
+    return((a - 1) * x**(-a))
+
```

Comment:
Thank you for your advice. Yes. You are absolutely right. But this process, inverse the CDF, is just equivalent and the speed is just the same as we vectorized that code.